multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/TimeoutAsyncFlowableStep.java

```diff
@@ -81,8 +78,9 @@ protected Duration calculateTimeout(ProcessContext context, TimeoutType timeoutT
                                                                   .get(timeoutType.getCommandLineAndGlobalLevelParamName()));
                 timeoutParameterName = timeoutType.getCommandLineAndGlobalLevelParamName();
             } else {
-                timeoutFinal = Duration.ofSeconds(timeoutGlobalLevelParameter.getDefaultValue()
-                                                                             .getSeconds());
+                timeoutFinal = Duration.ofSeconds(timeoutType.getCommandLineLevelVariable()
+                                                             .getDefaultValue()
+                                                             .getSeconds());
                 timeoutParameterName = DEFAULT_TIMEOUT;
             }
         }
```

Comment:
remove logTimeout method as it only does 1 thing

Reply:
or - 
 String timeoutTypeName = processString(timeoutType.getModuleLevelParamName())
 remove this line, move the processString invocation to the logTimeout method so that the method invocation now looks like:
 logTimeout(timeoutType.getModuleLevelParamName(), timeoutParameterName, timeoutFinal.toSeconds())